include/nbl/core/math/floatutil.h

```diff
@@ -537,6 +537,9 @@ inline rgb32f rgb19e7_to_rgb32f(uint64_t _rgb19e7)
 	return r;
 }
 
+uint32_t& floatBitsToUint(float& _f);
+uint32_t floatBitsToUint(float&& _f);
```

Comment:
we already have these?

Reply:
declaration, you can't skip it otherwise error

Reply:
basically because `floatBitsToUint` is defined in a header that includes `floatutil.h`, you can't include `floatBitsToUint`'s header then there and you have to do forward declaration

Reply:
then have a declaration of the functions in `core/decl/math/floatutil.h` and the definition in `core/def/math/floatutil.h` like other templates

Reply:
https://github.com/Devsh-Graphics-Programming/Nabla/issues/212